### DISCUSSION TITLE
Optionnal content groups (layers) in PDF file

### DISCUSSION THREAD

**Discussion by djs45** [py-pdf/pypdf#1653] [General]
Hello.
Is there a way to hide (visible or for printing) or isolate Optionnal content group (also named layer) in a PDF file ?
I have to export each layer of a PDF file to image.
If I can export each layer from a PDF file to multiple PDF file, it's OK.
Many thanks

Sébastien

**Comment by pubpub-zz** [comment]
@djs45,
can you share such a file to see what can PyPDF do currently?

**Comment by djs45** [comment]
Hello.
Here is a PDF with 3 layers. 
I needto export to 3 images or 3 PDF with isolated layer.
I need to have transparency background on each 3 images and/or PDF.
Thank You

[test_PDF.pdf](https://github.com/py-pdf/pypdf/files/10816170/test_PDF.pdf)
